torch/__init__.py

```diff
@@ -126,6 +126,26 @@
                 err.strerror += ' Error loading "{}" or one of its dependencies.'.format(dll)
                 raise err
 
+    try:
+        load_library_with_flags('vcruntime140.dll')
+        load_library_with_flags('msvcp140.dll')
+        if cuda_version not in ('9.2', '10.0'):
+            load_library_with_flags('vcruntime140_1.dll')
+    except OSError:
+        print('''Microsoft Visual C++ Redistributable is not installed, which may lead to the DLL load failure.
+                 It can be downloaded at https://aka.ms/vs/16/release/vc_redist.x64.exe''')
+
+    try:
+        load_library_with_flags('libiomp5md.dll')
+    except OSError:
+        print('''Intel OpenMP is not installed, which may lead to the DLL load failure. 
```

Comment:
libiomp5md.dll is part of pytorch wheel, why user still need to install Intel OpenMP?

Reply:
I guess, people aren't necessarily always going to be using our prebuilt Wheels. But that leads me to another question @peterjc123: is Intel OpenMP the *only* openmp configuration we support on Windows?

Reply:
I think it is the root cause for the issue https://github.com/pytorch/pytorch/issues/35803#issuecomment-682356457, but it doesn't seem to be that case. But right now, I'm trying to do sth. like @ezyang described.
As for OpenMP, actually we support VCOMP (MSVC Bulitin) and Intel OpenMP, but someone claims that PyTorch fails to import if `detectron` is built with VCOMP. We may look into that at some time.

Reply:
OK. So just to double check, this doesn't seem to fix #35803, so we should wait on it some more?

Reply:
@ezyang Let's wait for a while. This PR isn't that urgent, I think.

Reply:
@ezyang But according to the user's feedback in that issue, it seems that either `asmjit` or `fbgemm` is using VCOMP. I think we should use a unified choice for OMP (->Intel OMP).